.github/workflows/pr_comments.yml

```diff
@@ -0,0 +1,18 @@
+name: PR comments
+
+on:
+  issue_comment:
+    types: [created, updated]
+
+jobs:
+  pr_commented:
+    name: PR comment
+    if: ${{ github.event.issue.pull_request }}
```

Comment:
Only run when the comment in on a pull request.

I'll limit the run instructions further when I have a better idea what is available